### PR TITLE
Fix double-log from double-super invocation

### PR DESCRIPTION
**Pull request by jpr5**
Per issue #18.

### CODE REVIEW

**Comment by jpr5** [comment]
Passes now.

File: lib/puma/daemon/runner_adapter.rb [hunk 1]
```diff
@@ -35,9 +35,11 @@ def daemonize!
             end
 
             def log(str)
-              if super.respond_to?(:log)
-                super(str) unless str == 'Use Ctrl-C to stop'
-              else
+              return if str == 'Use Ctrl-C to stop'
```
Comment (kigster) [inline lib/puma/daemon/runner_adapter.rb:38]: Let's change this to a regular expression match:

```ruby
return if str =~ /Ctrl-C/
```

**Review by kigster — CHANGES_REQUESTED** [review]
Let's make this a regular expression check that would be more resilient to future upstream changes. Then I'll merge it. Thank you for your contribution!

**Comment by jpr5** [comment]
Alright.

**Review by kigster — APPROVED** [review]

**Pull request merged**